Week2/homework/maartjes-work.js

```diff
@@ -47,19 +47,23 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const maartjesTasks = tasks.map(tasks => tasks.duration / 60)
+  const filteredTasks = maartjesTasks.filter(taskHour => taskHour >= 2)
+  const totalEarnings = filteredTasks.reduce((earnings, taskHour) => earnings + taskHour * hourlyRate, 0);
+  return totalEarnings;
```

Comment:
👍 for logic. However, you are missing semi-colons at the end of line 50 and 51. Also, nice that you used meaningful names for the callback arguments.